com.stansassets.build/Editor/Steps/UnityPlayerBuildStep.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEditor.Build.Reporting;
+using UnityEditor.Callbacks;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace StansAssets.Build.Editor
+{
+    class UnityPlayerBuildStep : IBuildStep
+    {
+        public int Priority => m_Priority;
+
+        public string Name => m_Name;
+        private string m_Name = "StepName";
+
+        private int m_Priority = 0;
+        private static List<IBuildTask> s_Tasks;
+        private IBuildContext m_BuildContext;
+        private event Action<BuildStepResultArgs> m_OnCompleteCallback;
```

Comment:
Better to assign delegate {} to events inside a class to avoid null ref checks when calling it, like
`private event Action<BuildStepResultArgs> m_OnCompleteCallback = delegate {};`